File: app/src/main/java/at/priv/graf/zazentimer/Bell.java
package at.priv.graf.zazentimer;

import android.net.Uri;

public class Bell {
    private String name;
    private Uri uri;

    public Bell(Uri uri, String str) {
        this.uri = uri;
        this.name = str;
    }

    public String getName() {
        if (this.name.startsWith("bell_")) {
            return this.name.substring(5);
        }
        return this.name;
    }

    public Uri getUri() {
        return this.uri;
    }
}

File: app/src/main/kotlin/at/priv/graf/zazentimer/bo/Bell.kt
package at.priv.graf.zazentimer.bo

import android.net.Uri

data class Bell(
    var uri: Uri,
    private var bellName: String,
) {
    companion object {
        private const val BELL_PREFIX_LENGTH = 5
    }

    fun getName(): String = if (bellName.startsWith("bell_")) bellName.substring(BELL_PREFIX_LENGTH) else bellName
}
